package.json

```diff
@@ -70,7 +71,7 @@
     "universal-cookie": "4.0.4"
   },
   "peerDependencies": {
-    "@edx/paragon": "^7.1.3",
+    "@edx/paragon": "^12.8.0",
```

Comment:
Is it true that this is now the minimum paragon version that frontend-platform is compatible with?